<launch>
<!--- Depth image to laser scan -->
    <node pkg="depthimage_to_laserscan" type="depthimage_to_laserscan" name="depthimage_to_laserscan" output="screen">
        <param name="scan_height" value="40"/> 
        <param name="range_max" value="10"/> 
        <param name="output_frame_id" value="base_link"/> 
        <remap from="image" to="/pepper_robot/naoqi_driver/camera/depth/image_raw" />
        <remap from="camera_info" to="/pepper_robot/naoqi_driver/camera/depth/camera_info" />
    </node>
</launch>